### PR TITLE
Webadmin

### PR DESCRIPTION
**Pull request by artur9010**

### CODE REVIEW

**Comment by SphinxC0re** [comment]
Looks good. Merge if compiles

File: MCServer/webadmin/login_template.html [hunk 1]
```diff
@@ -51,17 +51,17 @@
 	<div class="upper">
 		<div class="wrapper">
 			<ul class="menu bottom_links">
-				<li><a href="http://www.mc-server.org" target="_blank">MCServer</a></li>
+				<li><a href="http://cuberite.org" target="_blank">Cuberite</a></li>
 				<li><a href="http://forum.mc-server.org" target="_blank">Forums</a></li>
 				<li><a href="http://builds.cuberite.org" target="_blank">Buildserver</a></li>
 				<li><a href="http://mc-server.xoft.cz/LuaAPI" target="_blank">API Documentation</a></li>
```
Comment (SphinxC0re) [inline MCServer/webadmin/login_template.html]: Oh, you should change this to the newly created domain api-docs.cuberite.org

File: MCServer/webadmin/template.lua [hunk 2]
```diff
@@ -177,17 +177,17 @@ function ShowPage(WebAdmin, TemplateRequest)
 	<div class="upper">
 		<div class="wrapper">
 			<ul class="menu bottom_links">
-				<li><a href="http://www.mc-server.org" target="_blank">MCServer</a></li>
+				<li><a href="http://cuberite.org" target="_blank">Cuberite</a></li>
 				<li><a href="http://forum.mc-server.org" target="_blank">Forums</a></li>
 				<li><a href="http://builds.cuberite.org" target="_blank">Buildserver</a></li>
 				<li><a href="http://mc-server.xoft.cz/LuaAPI" target="_blank">API Documentation</a></li>
```
Comment (SphinxC0re) [inline MCServer/webadmin/template.lua]: As told above

**Comment by SphinxC0re** [comment]
Can you rebase onto the current master and squash the commits into a single one? You also would have to update the commit message

File: MCServer/webadmin/template.lua [hunk 3]
```diff
@@ -177,17 +177,17 @@ function ShowPage(WebAdmin, TemplateRequest)
...
-				<li><a href="http://book.mc-server.org/" target="_blank">User's Manual</a></li>
+				<li><a href="http://api-docs.cuberite.org" target="_blank">API Documentation</a></li>
+				<li><a href="http://book.cuberite.org/" target="_blank">User's Manual</a></li>
 			</ul>
 		</div>
 	</div>
 	<div class="lower">
 		<div class="wrapper">
-			<span id="copyright">Copyright © <a href="http://www.mc-server.org" target="_blank">MCServer Team</a> 2014.</span>
+			<span id="copyright">Copyright © <a href="http://www.cuberite.org" target="_blank">Cuberite Team</a> 2015.</span>
```
Comment (bearbin) [inline MCServer/webadmin/template.lua]: Could you remove the date from the copyright notice, as it serves no purpose?

**Comment by madmaxoft** [comment]
Rebased and merged manually.

**Pull request closed without merging**